modeling-cmds/src/def_enum.rs

```diff
@@ -403,14 +407,35 @@ define_modeling_cmd_enum! {
             pub cylinder_id: Uuid,
             /// Number of revolutions.
             pub revolutions: f64,
-            /// Start angle (in degrees).
+            /// Start angle.
+            #[serde(default = "default_angle")]
```

Comment:
```suggestion
            #[serde(default)]
```
Angle already impls Default (and it returns 0 degrees), see [here](https://docs.rs/kittycad-modeling-cmds/latest/kittycad_modeling_cmds/shared/struct.Angle.html#impl-Default-for-Angle)